belgif-rest-problem/src/main/java/io/github/belgif/rest/problem/internal/JacksonUtil.java

```diff
@@ -24,9 +24,19 @@ private JacksonUtil() {
      * @return the BadRequestProblem
      */
     public static BadRequestProblem toBadRequestProblem(JsonMappingException e) {
-        String name = e.getPath().stream().map(Reference::getFieldName).collect(Collectors.joining("."));
+        StringBuilder name = new StringBuilder();
+        for (Reference reference : e.getPath()) {
+            if (reference.getFrom() != null && reference.getFrom() instanceof List) {
```

Comment:
Sonar [suggests to remove the null check](https://sonarcloud.io/project/issues?sinceLeakPeriod=true&issueStatuses=OPEN%2CCONFIRMED&pullRequest=177&id=belgif_rest-problem-java&open=AZTVTJO5vLz2aA1LC8Ep).